data-share/data-share-service/src/main/java/io/mosip/datashare/util/PolicyUtil.java

```diff
@@ -50,6 +52,15 @@ public class PolicyUtil {
 	@Autowired
 	private ObjectMapper mapper;
 
+	/** The static data share policy used for sharing the data. */
+	private static final String STATIC_DATA_SHARE_POLICY = "mosip.data.share.static.share.policy";
+
+	/** The static data share policy used for sharing the data. */
+	private static final String STATIC_DATA_SHARE_POLICY_ID = "mosip.data.share.static.share.policyid";
+
+	/** The static data share policy used for sharing the data. */
+	private static final String STATIC_DATA_SHARE_SUBSCRIBER_ID = "mosip.data.share.static.share.subscriberid";
+

```

Comment:
Can we not use the @value based reading of these property values ?

Reply:
Lets change the prefix for all the above 3 properties as below
1. mosip.data.share.static-policy.policy-json
2. mosip.data.share.static-policy.policy-id
3. mosip.data.share.static-policy.subscriber-id

Reply:
instead of isnull, can we use the isEmpty ? that way we check the empty string condition also